### PR TITLE
Ie11 overlaps svg#15221

### PR DESCRIPTION
**Pull request by sgrg1**
issue fixed

### CODE REVIEW

File: src/styles/partials/components/_card-container.scss [hunk 1]
```diff
@@ -9,10 +9,14 @@
...
-
-  .dg_icon-container {
-    color: $gold;
-    font-size: $largest-icon-size;
-    margin-bottom: 20px;
+}
+.dg_icon-container {
+  color: $gold;
+  font-size: $largest-icon-size;
+  margin-bottom: 20px;
```
Comment (martypowell) [inline src/styles/partials/components/_card-container.scss]: this should use a margin variable.

File: src/styles/partials/components/_card-container.scss [hunk 2]
```diff
@@ -9,10 +9,14 @@
...
+}
+.dg_card__content {
```
Comment (martypowell) [inline src/styles/partials/components/_card-container.scss]: was this added, if so why? I didn't see anything about this in the bug?

Comment (sgrg1) [reply src/styles/partials/components/_card-container.scss]: I believe it was from previous PR fix "change the font size to 18px". I must have created branch from that fix.

File: src/components/basic/Card.md [hunk 3]
```diff
@@ -47,15 +49,17 @@ import Section from "./containers/Section";
             role="img"
             aria-label="Placeholder: Thumbnail"
           >
-            <rect width="100%" height="200px" fill="#C2C2C2"></rect>
+            <rect width="100%" height="150px" fill="#C2C2C2"></rect>
```
Comment (martypowell) [inline src/components/basic/Card.md]: it looks like this placeholder svg is being repeated over and over. This sounds like a good instance to create a small component within this file and then reference that.

Example:

```jsx
const ImagePlaceholder = props => <svg><rect ></rect></svg>;
```

Then you can use it in the cards, like so

```jsx
<CardContent>
  <ImagePlaceHolder />
...
```



Then in the future, if we have to tweak the svg, we only need to do it in one place.

File: src/styles/partials/components/_card-container.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/partials/components/_card-container.scss]: no problem, i was hitting these backwards, we can wait until that one is completed

File: src/styles/partials/components/_card-container.scss (hunk 1 above)
Comment (sgrg1) [reply src/styles/partials/components/_card-container.scss]: done

File: src/components/basic/Card.md (hunk 3 above)
Comment (sgrg1) [reply src/components/basic/Card.md]: done

**Review by martypowell — APPROVED** [review]

**Pull request merged**